cigaradvisor/scripts/templates/blog-post.js

```diff
@@ -70,9 +71,7 @@ export default async function decorate(main) {
   const articleBlurb = getMetadata('articleblurb');
 
   if (!category) {
-    const url = window.location.href;
-    const regex = /\/cigaradvisor\/category\/([^/]+)/;
-    const match = url.match(regex);
+    const match = getCategory(window.location.href);
```

Comment:
can you review this logic: 
![image](https://github.com/hlxsites/famous-smoke-cigaradvisor/assets/53608019/8789fb9f-fffa-42ff-8600-e775cff8c022)

Reply:
@kailasnadh790 good catch ! What's the id of the default category ?

Reply:
Ah, `uncategorized` will push a fix